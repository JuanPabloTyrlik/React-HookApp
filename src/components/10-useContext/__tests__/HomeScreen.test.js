import { mount } from 'enzyme';
import React from 'react';
import { HomeScreen } from '../HomeScreen';
import { UserContext } from '../UserContext';

describe('Tests on HomeScreen', () => {
    const user = {
        id: 1,
        nombre: 'Juan Pablo',
    };
    const wrapper = mount(
        <UserContext.Provider value={{ user }}>
            <HomeScreen />
        </UserContext.Provider>
    );
    test('should render correctly', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
