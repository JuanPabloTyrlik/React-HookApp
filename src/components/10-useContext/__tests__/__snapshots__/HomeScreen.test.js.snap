// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests on HomeScreen should render correctly 1`] = `
Array [
  <h1>
    Home Screen
  </h1>,
  <hr />,
  <pre>
    {
    "id": 1,
    "nombre": "Juan Pablo"
}
  </pre>,
]
`;
